.github/workflows/notify-release.yml

```diff
@@ -7,20 +7,6 @@ on:
 
 jobs:
   checking-pending-prs:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v2
-      with:
-        node-version: lts/*
-    - name: Install Dependencies
-      run: npm ci
-    - name: Check for PRs not yet released
-      run: node ./bin/pending-prs.js
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL }}
-        SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
-        SLACK_SECRET: ${{ secrets.SLACK_SECRET }}
-
+    uses: newrelic/node-newrelic/.github/workflows/unreleased-prs.yaml@abstract-release-notification
```

Comment:
hardcoding branch in hopes I can test the workflow before we merge by clicking the button

Reply:
doesn't work to test from this branch. ill revert this

Reply:
oh actually i can use act. 

`act -j checking-pending-prs`